docs/source/support/new-assistant.rst

```diff
@@ -0,0 +1,136 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Moving to the new DAML assistant
+################################
+
+New command-line tool for working with the DAML SDK: DAML Assistant, or ``daml``. Many commands are similar to using the old SDK Assistant (``da``), but with some changes:
+
+- Simplified installation process
+- Consistent behaviour of the ``new`` command
```

Comment:
What exactly does “consistent” mean here?

Reply:
Yeah, this is a bit first-draft. Let me find what Fran's original notes said:

> The `new` command now takes arguments in a consistent order

I think that's better